lib/actions.js

```diff
@@ -1,109 +1,152 @@
+    copyFolderSync(path.join(templatesDir, template), projectName, [
+      path.join(templatesDir, template, "node_modules"),
+    ]);
   } else {
     // Execute git clone
-    console.log(`\n${colors.green('>')} Fetching Template from GitHub ✨\n\n`);
+    console.log(`\n${colors.green(">")} Fetching Template from GitHub ✨\n\n`);
     try {
-      await executeCommand('git', ['clone', template, projectName], {
-        stdio: [process.stdin, process.stdout, process.stderr]
+      await executeCommand("git", ["clone", template, projectName], {
+        stdio: [process.stdin, process.stdout, process.stderr],
       });
     } catch (err) {
       throw err;
     }
   }
 
   // remove package-lock.json if preferred installer is yarn
-  if (installer === 'yarn') {
-    fs.unlinkSync(path.join(projectName, 'package-lock.json'));
+  if (installer === "yarn") {
+    fs.unlinkSync(path.join(projectName, "package-lock.json"));
   }
 
   // Delete git history
-  rmdirRecursiveSync(path.join(projectName, '.git'));
+  rmdirRecursiveSync(path.join(projectName, ".git"));
 }
 
 async function main(projectName, options) {
   // if projectName is falsy, prompt user for new name
   if (!projectName) {
-    projectName = (await prompts({
-      type: 'text',
-      message: 'Enter Name of your project',
-      name: 'projectName',
-      initial: 'hello-world'
-    })).projectName;
+    projectName = (
+      await prompts({
+        type: "text",
+        message: "Enter Name of your project",
+        name: "projectName",
+        initial: "hello-world",
+      })
+    ).projectName;
   }
 
-  projectName = projectName.toLowerCase().replace(/ |_/g, '-');
+  projectName = projectName.toLowerCase().replace(/ |_/g, "-");
   const template = getTemplate(options.template);
   const installer = await getInstaller(options.installer);
-  
-  console.log(`\n${colors.green('>>')} Creating New Abell Project 🌻`) 
-  console.log(`${colors.cyan('Directory:')} ${projectName}\n${colors.cyan('Template:')} ${template}\n${colors.cyan('Package Installer:')} ${installer}\n`);
+
+  console.log(`\n${colors.green(">>")} Creating New Abell Project 🌻`);
+  console.log(
+    `${colors.cyan("Directory:")} ${projectName}\n${colors.cyan(
+      "Template:"
+    )} ${template}\n${colors.cyan("Package Installer:")} ${installer}\n`
+  );
 
   // Create project directory
-  await createProject(projectName, template, installer)
+  await createProject(projectName, template, installer);
 
   // Install dependencies
-  console.log(`\n${colors.green('>')} Installing Dependencies 📚\n\n`);
-  
-  const isWindows = /^win/.test(process.platform);
-  let installerCommand = [isWindows ? 'npm.cmd' : 'npm', ['install']];
-  if (installer === 'yarn') {
-    installerCommand = [isWindows ? 'yarn.cmd' : 'yarn'];
+  console.log(`\n${colors.green(">")} Installing Dependencies 📚\n\n`);
+
+  // checking if the template is local
+  if (template === "default" || template === "minimal") {
```

Comment:
This condition checks if the template is local or not. the local template can have template value `default` or `minimal`.